gordo_components/builder/build_model.py

```diff
@@ -240,12 +241,23 @@ def _build(self) -> Tuple[Optional[sklearn.base.BaseEstimator], dict]:
                 scaler = self.evaluation_config.get("scoring_scaler")
                 metrics_dict = self.build_metrics_dict(metrics_list, y, scaler=scaler)
 
+                # Generate metadata about CV train, test splits
+                tssplit = TimeSeriesSplit(n_splits=3)
+                for i, (train_ind, test_ind) in enumerate(tssplit.split(X)):
+                    prefix = f"fold-{i+1}"
+                    splits.update(
+                        {
+                            f"{prefix}_train_start": X.index[train_ind[0]].isoformat(),
+                            f"{prefix}_train_end": X.index[train_ind[-1]].isoformat(),
+                            f"{prefix}_test_start": X.index[test_ind[0]].isoformat(),
+                            f"{prefix}_test_end": X.index[test_ind[-1]].isoformat(),
+                            f"{prefix}_n_train": len(train_ind),
+                            f"{prefix}_n_test": len(test_ind),
+                        }
+                    )
+
```

Comment:
The method is already quite long, what do you think about putting this into a different method/function?